Sources/src/FromExtMonFNALMARSFile_source.cc

```diff
@@ -118,46 +118,46 @@ namespace mu2e {
   int marsToMu2eParticleCode(int marsPID) {
     static std::map<int,int> table;
     if(table.empty()) {
-      table[1] = +2212; // proton
-      table[2] = +2112; // neutron
-      table[3] = +211; // pi+
-      table[4] = -211; // pi-
-      table[5] = +321; // K+
-      table[6] = -321; // K-
-      table[7] = -13; // mu+
-      table[8] = +13; // mu-
-      table[9] =  22; // gamma
-      table[10] = -11; // e-
-      table[11] = +11; // e+
-      table[12] = -2212; // p- (antiproton)
-      table[13] = 111; // pi0
-      table[14] = 1000010020; // deutron
-      table[15] = 1000010030; // H3
-      table[16] = 1000020030; // He3
-      table[17] = 1000020040; // He4
-      table[18] = +14; // numu
-      table[19] = -14; // numubar
-      table[20] = +12; // nue
-      table[21] = -12; // nuebar
-      table[22] = 130; // K0L
-      table[23] = 310; // K0S
+      table[1] = PDGCode::proton; // proton
+      table[2] = PDGCode::n0; // neutron
+      table[3] = PDGCode::pi_plus; // pi+
+      table[4] = PDGCode::pi_minus; // pi-
+      table[5] = PDGCode::K_plus; // K+
+      table[6] = PDGCode::K_minus; // K-
+      table[7] = PDGCode::mu_plus; // mu+
+      table[8] = PDGCode::mu_minus; // mu-
+      table[9] = PDGCode::gamma; // gamma
+      table[10] = PDGCode::e_plus; // e-
+      table[11] = PDGCode::e_minus; // e+
```

Comment:
Another case of e+/e- swap.   Perhaps this code should be factored out (as another PR).